### PR TITLE
fix: Misc QA fixes

### PR DESCRIPTION
**Pull request by rikkekantega**
- Use button component on "Utvid kart"
- Update message component to obey design system on contrast bg/text for error messages 
- Minor text adjustments ticket view
- Accessible ticket count controls
- Logo click label and "navigate to start" action consistant with user preference
- Make tab bar height adjust to font scaling, and disappearingheader content height calculation use responsive tab bar height
- Make screen readers ignore icon picker, as it's currently not operatable by using screen reader navigation 

2x font resizing tab bar:
image

### CODE REVIEW

File: src/utils/navigation.tsx [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+  Preference_ScreenAlternatives,
+} from '../preferences';
+import {TabNavigatorParams} from '../navigation/TabNavigator';
+import {useWindowDimensions} from 'react-native';
 
-const HOME_TAB_NAME = 'Assistant';
-
-export const useNavigateHome = () => {
+export const useBottomNavigationStyles = (): {height: number} => {
+  const {fontScale} = useWindowDimensions();
```
Comment (mikaelbr) [inline src/utils/navigation.tsx:12]: Hey! I didn't know about that! I used `DeviceInfo` which is way more work! Thank you!

**Review by mikaelbr — APPROVED** [review]
Looks good! There's been some feedback about the "click on the logo" functionality, where clicking it expects to "reset the view" (we've also discussed this previously). But that is probably a different discussion than this QA fix.

**Review by cbrevik — APPROVED** [review]
Nice! Lots of good fixes here 🚀

**Comment by rikkekantega** [comment]
> Looks good! There's been some feedback about the "click on the logo" functionality, where clicking it expects to "reset the view" (we've also discussed this previously). But that is probably a different discussion than this QA fix.

I agree, we need to work out a better logo look and function.

**Pull request merged**